lib/suspenders/generators/production/manifest_generator.rb

```diff
@@ -0,0 +1,22 @@
+require "rails/generators"
+
+module Suspenders
+  module Production
+    class ManifestGenerator < Rails::Generators::Base
+      source_root File.expand_path(
+        File.join("..", "..", "..", "..", "templates"),
+        File.dirname(__FILE__),
+      )
+
+      def render_manifest
+        template "app.json.erb", "app.json"
+      end
+
+      private
+
+      def app_name
```

Comment:
In a future PR I'm going to extract `app_name` and the `source_root` to a common base class.